app/src/main/java/com/example/nsc_events/data/Event.kt

```diff
@@ -2,10 +2,38 @@ package com.example.nsc_events.data
 
 import java.util.Date
 
+data class SocialMedia(
+    val facebook: String,
+    val twitter: String,
+    val instagram: String,
+    val hashtag: String
+)
+
 data class Event(
-    val eventName: String,
-    val description: String,
+    val eventTitle: String,
+    val eventDescription: String,
     /* TODO: change date variable to a Date data type instead of string */
-    val date: String
+    val eventCategory: String,
+    val date: Date,
+    val eventStartTime: Date,
+    val eventEndTime: Date,
```

Comment:
I believe eventStartTime and eventEndTime should be changed to String type to match our schema https://github.com/SeattleColleges/nsc-events-nestjs/blob/main/src/activity/schemas/activity.schema.ts